tests/test_logging.py

```diff
@@ -133,3 +138,34 @@ class App:
     # dir_contents = tmpdir.listdir()
     # assert len(dir_contents) == 1
     # assert dir_contents[0].basename == 'foo.json'
+
+
+@pytest.mark.parametrize('service_id', ["fake-service_id", None])
+def test_logging_records_statsd_stats(app_with_statsd, service_id):
+    app = app_with_statsd
+    statsd = app_with_statsd.statsd_client
+    logging.init_app(app_with_statsd, statsd)
+
+    @app.before_request
+    def record_request_details():
```

Comment:
We've got this on the API https://github.com/cds-snc/notification-api/blob/3db95eb950de58faeecefe0818c95c7872863d1f/app/__init__.py#L234-L237